system-tests/projects/vueclivue2-configured/package.json

```diff
@@ -10,8 +10,6 @@
     "vue": "^2.6.11"
   },
   "devDependencies": {
-    "@cypress/vue2": "file:../../../npm/vue2",
```

Comment:
This doesn't do what you expect on windows apparently. Also, seems like it isn't strictly required.